util/fipstools/acvp/acvptool/subprocess/subprocess.go

```diff
@@ -106,7 +106,6 @@ func NewWithIO(cmd *exec.Cmd, in io.WriteCloser, out io.ReadCloser) *Subprocess
 		"ctrDRBG":           &drbg{"ctrDRBG", map[string]bool{"AES-128": true, "AES-192": true, "AES-256": true}},
 		"hmacDRBG":          &drbg{"hmacDRBG", map[string]bool{"SHA-1": true, "SHA2-224": true, "SHA2-256": true, "SHA2-384": true, "SHA2-512": true}},
 		"KDF":               &kdfPrimitive{},
-		"KAS-KDF":           &hkdf{},
```

Comment:
In BoringSSL's commit, the line below has `&hkdf{},` do we want to have the same? See https://github.com/google/boringssl/commit/726585c58ec89f65ab669187665232790cbf3d02#diff-6fae1d42d6f42e954a91feb6e08d8802c80eb49863f45a76ed5d1ae84414398cR109

Reply:
I think at the very least the change will reduce confusion as to what is actually being tested with the KDF algorithm. The current name `kdfPrimitive` could infer any one of the different KDFs that ACVP supports when it actually is just `HKDF`. Will change.

Reply:
Does this bring the file `hkdf.go` to be similar to BoringSSL's?, if so, may be we should update the PR description and maybe mention in in the commit message at the time of merge.

Reply:
I would argue that BoringSSL's change brought them in line with _us_ :) but yes their version and ours are almost the same at this point. I'll make a note in the description and make sure to mention it in the commit message at time of merge as well.